substrate/frame/support/test/tests/pallet.rs

```diff
@@ -151,6 +153,7 @@ pub mod pallet {
 
 		type Balance: Parameter + Default + TypeInfo;
 
+		#[allow(deprecated)]
```

Comment:
I do we need to allow this associated type to be deprecated?

Reply:
references deprecated `Event`

Reply:
ah yes thanks